vendor/sxd-xpath-no-unsafe/.github/workflows/ci.yml

```
@@ -0,0 +1,66 @@
+on: [push, pull_request]
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)